src/uct/ib/rc/base/rc_ep.c

```diff
@@ -200,12 +200,7 @@ ucs_status_t uct_rc_ep_pending_add(uct_ep_h tl_ep, uct_pending_req_t *n,
                                    unsigned flags)
 {
     uct_rc_iface_t *iface = ucs_derived_of(tl_ep->iface, uct_rc_iface_t);
-    uct_rc_ep_t *ep = ucs_derived_of(tl_ep, uct_rc_ep_t);
-
-    if (uct_rc_ep_has_tx_resources(ep) &&
-        uct_rc_iface_has_tx_resources(iface)) {
-        return UCS_ERR_BUSY;
-    }
+    uct_rc_ep_t *ep       = ucs_derived_of(tl_ep, uct_rc_ep_t);
```

Comment:
func logic is broken now:
on line 210 we assume that there is no iface resources based on the check you removed